core/src/playitgg/errors.rs

```diff
@@ -0,0 +1,72 @@
+/* 
+Copyright 2022 Developed Methods LLC
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
```

Comment:
May need to reproduce the copyright notice in our repo

Reply:
I think it should be fine since we only use source code and it's credited anywhere it's used.